### PR TITLE
Bug 1154642 - [System] focus to corrent element and wait for transitione...

### PR DESCRIPTION
**Pull request by huchengtw-moz**
...nd before changing focus

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d314d9cd25641cb73e7867e8c56f9369e303bf40

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=59b2b0244e6b3e38852306ad01b87e7e6a4b9308

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c773166ebbe15809e290c89845b56b5600b334fb

File: apps/system/js/context_menu_view.js [hunk 1]
```diff
@@ -125,6 +123,29 @@
...
   };
 
+  ContextMenuView.prototype.focus = function() {
+    setTimeout(function() {
+      document.activeElement.blur();
+      this.elements.cancel.focus();
+    }.bind(this));
+  };
+
+  ContextMenuView.prototype._requestFocus = function() {
```
Comment (huchengtw-moz) [inline apps/system/js/context_menu_view.js]: I had add this function and move the 99 and 86, which are Service.request('focus');`, to here.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a4a8abbd83d1b29659af7c577ed9a8604374d5ab

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9e80921044b57b0a17552209406d482015b4cae4

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=94857238baeb6ab493774528862e5f32026304cc

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=52acc45fd98c7cd8d13311ffb46fb65276d5224c

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ddab00fde5ce40c6e2d3b07e9c726ae4151d51cb

**Pull request merged**